shared/test/testdata/subgraphConfigGenerator.ts

```diff
@@ -0,0 +1,80 @@
+import * as fs from 'node:fs';
+import * as process from 'node:process';
+import { federateSubgraphs, Subgraph } from '@wundergraph/composition';
+import { parse } from 'graphql';
+import { printSchemaWithDirectives } from '@graphql-tools/utils';
+
+function replacer<K, V>(key: K, value: V) {
```

Comment:
You could put this in a util file and call it from all tests. I think we can remove the pre build step.